pkg/apis/hub/v1alpha1/ai_service.go

```diff
@@ -50,9 +50,15 @@ type AIServiceSpec struct {
 
 // +k8s:deepcopy-gen=true
 
+type Secret struct {
+	SecretName string `json:"secretName"`
```

Comment:
Maybe we could simply call it `Name` WDYT?
It could be a good idea to add length validation such as `// +kubebuilder:validation:MaxLength=253` for example.

Reply:
As the parent configuration is Token or ApiKey, I think it's better to keep secretName

Reply:
I still think we should add a length validation